From 5bec2dca0ee9fe69dce9e454b50d54d446367e3b Mon Sep 17 00:00:00 2001
From: KENZ <KENZ.gelsoft@gmail.com>
Date: Sat, 13 May 2023 12:03:03 +0900
Subject: [PATCH 10/13] Workaround mio build error by disabling tests

- mio requires epoll() or kqueue() which is not available on Haiku.
---
 haiku_mozconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/haiku_mozconfig b/haiku_mozconfig
index 53092f20401c..95dc092f1847 100644
--- a/haiku_mozconfig
+++ b/haiku_mozconfig
@@ -1,4 +1,5 @@
 ac_add_options --enable-default-toolkit=cairo-gtk3-wayland-only
+ac_add_options --disable-tests
 ac_add_options --with-system-nspr
 ac_add_options --with-system-nss
 ac_add_options --without-wasm-sandboxed-libraries
-- 
2.37.3

